modules/ROOT/pages/errors/gql-errors/22N20.adoc

```diff
@@ -1,4 +1,12 @@
 = 22N20
 
 == Status description
-error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ $dim1 }D` coordinate reference system (CRS) and `{ $value }` coordinates. Use the equivalent `{ $dim2 }D` coordinate reference system instead.
+error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }` coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }` coordinate reference system instead.
```

Comment:
```suggestion
error: data exception - invalid spatial value dimensions. Cannot create POINT with `{ <<dim>>1 }D` coordinate reference system (CRS) and `{ <<value>> }` coordinates. Use the equivalent `{ $dim2 }D` coordinate reference system instead.
```

Looks like the D:s have fallen out, the parameter is e.g. 2 or 3 so it becomes 2D or 3D